core/src/main/java/org/infinispan/remoting/transport/jgroups/ClusterView.java

```diff
@@ -0,0 +1,61 @@
+package org.infinispan.remoting.transport.jgroups;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.infinispan.commons.util.Immutables;
+import org.infinispan.remoting.transport.Address;
+
+public class ClusterView {
+   static final int INITIAL_VIEW_ID = -1;
+   static final int FINAL_VIEW_ID = Integer.MAX_VALUE;
+
+   private final int viewId;
+   private final List<Address> members;
+   private final Set<Address> membersSet;
```

Comment:
`Transport.getMembers()` is part of the API, and applications may rely in `Transport.getMembers().get(0)` to return the coordinator. We may want to change that in the future, but definitely not in a minor version...